Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
@chrmoritz good idea or not a good idea, the fact remains that the grafana formula works currently and won't after this PR, at least as-is.